src/regtech_cleanup_api/entities/repos/institution_repo.py

```diff
@@ -6,28 +7,38 @@
 )
 from sqlalchemy.orm import Session
 from sqlalchemy import text
+from regtech_cleanup_api.entities.repos import repo_utils
+
+logger = logging.getLogger(__name__)
 
 
 def delete_domains_by_lei(session: Session, lei: str) -> List[FinancialInstitutionDomainDao] | None:
-    session.query(FinancialInstitutionDomainDao).filter(FinancialInstitutionDomainDao.lei == lei).delete()
-    session.commit()
-    return {"OK": True}
+    repo_utils.delete_helper(session, FinancialInstitutionDomainDao, lei)
```

Comment:
Since I can't see the repo_utils.delete_helper, I'm curious if this works because the previous "delete_helper" filter by table_obj.id but that's not what we're trying to delete here with FinancialInsitutionDomainDao.